1_OCaml/TP/TP2.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            (exp_rapide a (n/2) )**2.
</pre><p>Écrire <code>let b = exp_rapide a (n/2) ) in b*b</code>  plutôt que <code>exp_rapide a (n/2) )**2</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>